src/test/java/core/basesyntax/strategy/PurchaseOperationHandlerTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Transaction;
+import java.util.NoSuchElementException;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class PurchaseOperationHandlerTest {
+    private static Storage fruitStorage;
+    private static OperationHandler handler;
+    @Rule
+    public ExpectedException exceptionRule = ExpectedException.none();
+
+    @BeforeClass
+    public static void setup() {
+        fruitStorage = new Storage();
+        handler = new PurchaseOperationHandler();
+    }
+
+    @Test
+    public void purchase_operation_fruit_absent_exception() {

```

Comment:
maybe we add Ok or NotOk in the end of the method name?

Reply:
i had some eror with too long method names. i'm not sure.